open-science-training.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: default
+title: Open Science Training
+description: Online mentoring and training programs on open research practices.
+image: https://images.unsplash.com/photo-1542601906990-b4d3fb778b09
+photos:
+  name: Noah Buscher
+  license: CC BY-SA 4.0
+  url: https://images.unsplash.com/photo-1542601906990-b4d3fb778b09
+---
+
+<!-- <h1><figure class="image is-32x32"><img src="{% link images/three-pillars-icons/open-science-training-icon.png %}"></figure>Open Science Training</h1> -->
+
+Online mentoring and training programs for individuals and teams worldwide to learn about and adopt open research practices.
+
+## What makes us different:
+- Curriculum development & pedagogical expertise specifically in teaching researchers.
+- Our programs are built with a global community of experienced community builders and open practitioners.
+
+# Current work streams:
+- [Open Seeds (16-weeks)]({% link openseeds/index.md %})
```

Comment:
It could be 2 box there as for the index with the pillar with the logo and program name below

Reply:
💭 That could work...and look really good.

I am thinking that for the sake of consistency, would we have to do the same for all 3 Pillars pages?
If yes, would the cards not be too much.

Reply:
I would say no for now for the other pages because different projects do not have dedicated pages

Reply:
Sounds good. I'll put them in boxes now.